candle-nn/src/activation.rs

```diff
@@ -6,8 +6,7 @@ use serde::Deserialize;
 pub enum Activation {
     #[default]
     Gelu,
-    #[serde(rename = "gated-gelu")]
-    NewGelu,
+    ApproximateGelu,
```

Comment:
Please do not change the name, it's a bad name but in sync with pytorch and changing it breaks the api compatibility.

Reply:
Got it. I reverted it. Thanks